typings/codeceptjs.d.ts

```diff
@@ -507,6 +507,11 @@ declare namespace CodeceptJS {
     addNote(): void,
 
   }
+
+  export interface removeAnOrderPage {
+    selectOrderToRemove(): void,
+    enterReasonToRemoveOrder(): void
+  }
```

Comment:
Is this needed? I'm not sure it's used

Reply:
Not sure either, @KiranVarma02 did you add this?

Reply:
removed.